protoc-gen-grpc-gateway/internal/gengateway/template.go

```diff
@@ -568,14 +568,14 @@ func Register{{$svc.GetName}}{{$.RegisterFuncSuffix}}Server(ctx context.Context,
 	{{range $m := $svc.Methods}}
 	{{range $b := $m.Bindings}}
 	{{if or $m.GetClientStreaming $m.GetServerStreaming}}
-	mux.Handle({{$b.HTTPMethod | printf "%q"}}, pattern_{{$svc.GetName}}_{{$m.GetName}}_{{$b.Index}}, func(w http.ResponseWriter, req *http.Request, pathParams map[string]string) {
+	mux.Handle({{$b.HTTPMethod | printf "%q"}}, pattern_{{$svc.GetName}}_{{$m.GetName}}_{{$b.Index}}, "/{{$svc.File.GetName}}.{{$svc.GetName}}/{{$m.GetName}}", func(w http.ResponseWriter, req *http.Request, pathParams map[string]string) {
```

Comment:
Instead of adding this here, why don't we do add this as a parameter to `runtime.AnnotateContext` on line 586 and add it in there?

Reply:
@johanbrandhorst good idea, let me try.